### ISSUE TITLE
508-defect-3: Links should be styled as links, view your VA payment history

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#50631]
## Point of contact

@joshkimux

## Severity level

3, Moderate. Should be fixed in 1-3 sprints post-launch.

## Details

The "view your payment history" link is styled as a `button` which is materially dishonest. This causes several issues:
- It's materially dishonest and damages habituation on VA.gov 
- Voice assistive tech users may say "Click go to xyz _button_" which may result in failure as it is tagged as a `a`
- Screen reader users with low vision may become confused at a button being announced as a link, and even more so when navigating by links/buttons using the rotor
- Sighted users may not know that they can open this in a new tab because it looks like a button, especially on mobile which gives a stronger affordance for a tap vs. a tap/hold

This is a sitewide long-tail effort. [To better track button/link inconsistencies, please view this Mural.](https://app.mural.co/invitation/mural/vsa8243/1610052905994?sender=u2134c22982ad9c7b41798011&key=87284ca2-fd99-431d-b086-5890abf480ac).

<img width="739" alt="Screen Shot 2022-12-12 at 3 24 41 PM" src="https://user-images.githubusercontent.com/14154792/207147414-9be53305-0084-44b1-bc13-08ba8af54b1b.png">

## Reproduction steps

1. Log in as user 228
2. [Go to the view your VA payment page](https://staging.va.gov/va-payment-history/?postLogin=true)
3. Right click on the green "view your payment history" button and confirm it is a link

## Proposed solution or next steps

Use the action link instead of the green primary button

## References, articles, or WCAG support

* [Adam Silver: But sometimes buttons look like links](https://adamsilver.io/articles/but-sometimes-buttons-look-like-links/)
* [WCAG Success Criterion 3.2.4: Consistent Identification](https://www.w3.org/WAI/WCAG21/Understanding/consistent-identification.html)

## Type of issue

- [X] Screenreader
- [X] Keyboard
- [ ] Focus
- [ ] Headings
- [ ] Color
- [ ] Zoom
- [X] Semantics
- [ ] Axe-core
- [ ] Something else
